tests/routes/adage/v1/confirm_prebooking_test.py

```diff
@@ -18,6 +19,12 @@
 class Returns200Test:
     @freeze_time("2021-10-15 09:00:00")
     def test_confirm_prebooking(self, app, db_session) -> None:
+        redactor = EducationalRedactorFactory(
+            civility="Mme",
+            firstName="Jeanne",
+            lastName="Dodu",
+            email="jeanne.dodu@academie.ac",
```

Comment:
```suggestion
            email="jeanne.dodu@example.com",
```

Je ne sais plus quelle RFC c'est, mais normalement dans les tests nous sommes censés utiliser `@example.com`